mix.exs

```diff
@@ -10,55 +11,82 @@ defmodule OpenApiSpex.Mixfile do
       elixir: "~> 1.7",
       elixirc_paths: elixirc_paths(Mix.env()),
       start_permanent: Mix.env() == :prod,
-      description: description(),
+      consolidate_protocols: Mix.env() != :test,
       package: package(),
       deps: deps(),
-      consolidate_protocols: Mix.env() != :test,
-      source_url: "https://github.com/open-api-spex/open_api_spex",
-      homepage_url: "https://github.com/open-api-spex/open_api_spex",
-      docs: [extras: ["README.md"], main: "readme", source_ref: "v#{@version}"],
-      dialyzer: [
-        plt_add_apps: [:mix, :jason, :poison],
-        plt_add_deps: :apps_direct,
-        flags: ["-Werror_handling", "-Wno_unused", "-Wunmatched_returns", "-Wunderspecs"],
-        remove_defaults: [:unknown]
-      ]
+      docs: docs(),
+      dialyzer: dialyzer()
     ]
   end
 
   defp elixirc_paths(:test), do: ["lib", "test/support"]
   defp elixirc_paths(_), do: ["lib"]
 
-  # Run "mix help compile.app" to learn about applications.
   def application, do: [extra_applications: []]
 
-  defp description() do
-    "Leverage Open Api Specification 3 (swagger) to document, test, validate and explore your Plug and Phoenix APIs."
-  end
-
   defp package() do
     [
       name: "open_api_spex",
-      files: ["lib", "mix.exs", "README.md", "LICENSE", "CHANGELOG.md", ".formatter.exs"],
+      description:
+        "Leverage Open Api Specification 3 (swagger) to document, " <>
+          "test, validate and explore your Plug and Phoenix APIs.",
+      files: [
+        "lib",
+        "mix.exs",
+        "README.md",
+        "LICENSE",
+        "CHANGELOG.md",
+        "CONTRIBUTING.md",
+        "ROADMAP.md",
+        ".formatter.exs"
+      ],
       maintainers: [
         "Mike Buhot (m.buhot@gmail.com)",
         "Moxley Stratton (moxley.stratton@gmail.com)",
         "Pierre Fenoll (pierrefenoll@gmail.com)"
       ],
-      licenses: ["Mozilla Public License, version 2.0"],
-      links: %{"GitHub" => "https://github.com/open-api-spex/open_api_spex"}
+      licenses: ["MPL-2.0"],
```

Comment:
Is there a reason to prefer the acronym format here?

Reply:
See https://hex.pm/docs/publish.

> A list of licenses the project is licensed under. This attribute is required. It is recommended to use SPDX License identifier.

While it's not compulsory, it's recommended to use SPDX license identifier to set the right license.

For example, if we look at the list, https://spdx.org/licenses/, there are two types of Mozilla Public License 2.0:

- Mozilla Public License 2.0 - (MPL-2.0)
- Mozilla Public License 2.0 (no copyleft exception) - (MPL-2.0-no-copyleft-exception)

Setting the license to `Mozilla Public License, version 2.0` does not tell us which version being used.

Reply:
Thanks, TIL 👍